project-base/storefront/components/Layout/Header/MenuIconic/MenuMyAccountList.tsx

```diff
@@ -0,0 +1,84 @@
+import { MenuIconicSubItemLink } from './MenuIconicElements';
+import { SalesRepresentative } from './SalesRepresentative';
+import { EditIcon } from 'components/Basic/Icon/EditIcon';
+import { ExitIcon } from 'components/Basic/Icon/ExitIcon';
+import { SearchListIcon } from 'components/Basic/Icon/SearchListIcon';
+import { UserIcon } from 'components/Basic/Icon/UserIcon';
+import { useDomainConfig } from 'components/providers/DomainConfigProvider';
+import { TIDs } from 'cypress/tids';
+import useTranslation from 'next-translate/useTranslation';
+import { useCurrentCustomerUserPermissions } from 'utils/auth/useCurrentCustomerUserPermissions';
+import { useLogout } from 'utils/auth/useLogout';
+import { getInternationalizedStaticUrls } from 'utils/staticUrls/getInternationalizedStaticUrls';
+
+export const MenuMyAccountList: FC = () => {
+    const { t } = useTranslation();
+    const logout = useLogout();
+    const { canManageUsers } = useCurrentCustomerUserPermissions();
+    const { url } = useDomainConfig();
+    const [customerOrdersUrl, customerComplaintsUrl, customerEditProfileUrl, customerUsersUrl] =
+        getInternationalizedStaticUrls(
+            ['/customer/orders', '/customer/complaints', '/customer/edit-profile', '/customer/users'],
+            url,
+        );
+
+    const userMenuItemTwClass =
+        'h-14 rounded-xl bg-backgroundAccentLess border border-background hover:bg-background hover:border-borderAccentLess';
+    const userMenuItemIconTwClass = 'flex flex-row w-[44px] justify-start';
```

Comment:
is this necessary when we just wrapping the icon?

Reply:
Created `IconWrapper` and simplified the class names.